packages/core/package.json

```diff
@@ -73,6 +73,7 @@
     "iter-tools": "^7.5.0",
     "js-interpreter": "2.2.0",
     "lodash": "^4.17.21",
+    "mixpanel": "0.17.0",
```

Comment:
Caret?

Reply:
Ech, I don't like adding the caret so that it won't accidentally get upgraded to some other version. Should I add it back in?

Reply:
I mean, preventing accidental upgrades to another version is what `yarn.lock` is for, right?  Generally our practice has been not to pin unless there's a specific reason to.  Makes things harder for Dependabot, after all! :)